src/src/main.java

```diff
@@ -0,0 +1,121 @@
+import java.util.Scanner;
+
+public class main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        String gesuchtesWort = scanner.next();
+        String sternchen = new String(new char[gesuchtesWort.length()]).replace("\0", "*");
+        int versuche = 0;
+
+        while (versuche < 7 && sternchen.contains("*")) {
+            System.out.println("Versuche einen Buchstaben zu erraten!");
+            System.out.println(sternchen);
+            String buchstabe = scanner.next();
+
+
+            String neueSternchen = "";
+            for (int i = 0; i < gesuchtesWort.length(); i++) {
+                if (gesuchtesWort.charAt(i) == buchstabe.charAt(0)) {
+                    neueSternchen += buchstabe.charAt(0);
+                } else if (sternchen.charAt(i) != '*') {
+                    neueSternchen += gesuchtesWort.charAt(i);
+                } else {
+                    neueSternchen += "*";
+                }
+            }
+
+            if (sternchen.equals(neueSternchen)) {
+                versuche++;
+
+            } else {
+                sternchen = neueSternchen;
+            }
+            if (sternchen.equals(gesuchtesWort)) {
+                System.out.println("Richtig, du hast gewonnen " + gesuchtesWort);
+            }
+
+            if (versuche == 1) {
```

Comment:
cool 👍 

die ganzen versuchs-zweige könnten in else-if's verschachtelt werden, oder du könntest ein switch verwenden.